dev-tools/mage/gotest.go

```diff
@@ -168,7 +168,19 @@ func DefaultTestBinaryArgs() TestBinaryArgs {
 // Use RACE_DETECTOR=true to enable the race detector.
 // Use MODULE=module to run only tests for `module`.
 func GoTestIntegrationForModule(ctx context.Context) error {
-	module := EnvOr("MODULE", "")
+	modules := EnvOr("MODULE", "")
+	moduleArr := strings.Split(modules, ",")
```

Comment:
Let's print a warning if `modulesArr` is empty

Reply:
done in https://github.com/elastic/beats/pull/40053/commits/2ac881f9a29150250b27509fa57058960ad88697